packages/open-api-gateway/src/project/samples/typescript.ts

```diff
@@ -90,17 +97,15 @@ export class SampleApi extends Api {
       authType: AuthorizationType.IAM,
       integrations: {
         sayHello: {
-          function: new NodejsFunction(scope, "SayHelloHandler", {
-            entry: path.join(__dirname, "say-hello.handler.ts"),
-          }),
+          function: new NodejsFunction(scope, "say-hello"),
```

Comment:
no entry?

Reply:
If you don't specify an entry it infers it based on the id - this is where it checks for either `.ts` or `.js` which means this works whether it's a library in a monorepo (ie compiled to `.js`) or the main cdk target (ie not compiled).

Reply:
snazzy ;)